src/operations/copyTo.ts

```diff
@@ -42,7 +42,9 @@ export function copyTo(
   const { column, row } = origin;
 
   if (source.colorModel !== target.colorModel) {
-    throw new Error('Source and target should have the same color model.');
+    throw new Error(
+      'copyTo: source and target must have the same color model.',
+    );
```

Comment:
Any reason for including the name of the function in the error message? this is not a common pattern in JS, since the name is always visible in the first line of the stack trace.

Reply:
I don't know why, but I have been using this syntax a lot in the whole project, would you like me to remove all the function names? I think some of the already existing messages were like that and I replicated it.

Do we have a standard for whether the error messages should start with a lower or uppercase?

What should I do with the validators error messages, which contains the `process` parameter?

Reply:
> would you like me to remove all the function names? 

I think we should make an issue for this, yes

> Do we have a standard for whether the error messages should start with a lower or uppercase?

It should preferably start with an uppercase

> What should I do with the validators error messages, which contains the process parameter?

Remove the parameter.

Reply:
Ok, I'll create an issue then

Reply:
https://github.com/image-js/image-js-typescript/issues/192